pkg/segment/tracing/handler/tracehandler.go

```diff
@@ -537,7 +535,10 @@ func MakeTracesDependancyGraph() map[string]map[string]int {
 	return dependencyMatrix
 }
 
-func writeDependencyMatrix(dependencyMatrix map[string]map[string]int) {
+func writeDependencyMatrix(dependencyMatrix map[string]map[string]int, isAutoGenerated bool) {
+	if !isAutoGenerated {
+		return
+	}
```

Comment:
Why did you add the `isAutoGenerated` parameter if the function never does useful work when that is `false`?

Reply:
I removed this, i actually passed false for the `/generate-dep-graph` but removed it later. We don't need this variable.